wiki/en/Running-a-Server.md

```diff
@@ -52,31 +52,31 @@ Most people run Jamulus on a 3rd party/cloud host as a "headless" Server (no vid
 
 ---
 
-**Note** The following steps will result in you running an "Unregistered" Server. For more information on server types, [see below](#server-modes).
+**Note** The following steps will result in you running an "Unregistered" Server. For more information on Server types, [see below](#server-modes).
 
 ---
 
-To run a headless server on Linux, the following steps assume you are familiar with the command line and Debian/Ubuntu or similar distribution that uses systemd. 
+To run a headless server on Linux, the following steps assume you are familiar with the command line and Debian/Ubuntu or similar distribution that uses systemd.
 
 If you want to run a Server on a **Raspberry Pi** (or a different armhf/arm64 debian-based device), you will need to download the [latest armhf .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }}) or [latest arm64 .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-arm64 }}) depending on your CPU - not the default `amd64` ones for use on Intel/AMD machines.
```

Comment:
```suggestion
If you want to run a Server on a **Raspberry Pi** (or a different armhf/arm64 Debian-based computer), you will need to download the [latest armhf .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-armhf }}) or [latest arm64 .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-headless-arm64 }}) depending on your CPU - not the default `amd64` ones for use on Intel/AMD machines.
```

Reply:
Most listed Servers run Linux. The cheapest Ubuntu on AWS is ARM64. How do we know that most people installing headless Servers need AMD64?